test/e2e/network/service.go

```diff
@@ -100,18 +100,19 @@ var _ = SIGDescribe("Services", func() {
 	// TODO: We get coverage of TCP/UDP and multi-port services through the DNS test. We should have a simpler test for multi-port TCP here.
 
 	/*
-	   Testname: service-kubernetes-exists
-	   Description: Make sure kubernetes service does exist.
+		Release : v1.9
+		Testname: Kubernetes Service
+		Description: By default when a kubernetes cluster is running there SHOULD be a ‘kubernetes’ service running in the cluster.
```

Comment:
No, there MUST be. The test description below is not using "should" in the RFC 2119 sense.

Reply:
Fixed s/should/MUST